tasks/chkrootkit.yml

```diff
@@ -0,0 +1,5 @@
+- name: Add chkrootkit config file
+  copy: src=chkrootkit.conf dest=/etc/chkrootkit.conf owner=root group=root mode=644
```

Comment:
Can you please use the newer dictionary syntax here?  The attributes `owner`, `group` and `mode` are the defaults anyway, so you can omit them (up to you).
```
copy:
  src: chkrootkit.conf
  dest: /etc/chkrootkit.conf
```